pkg/pillar/devicenetwork/dnc.go

```diff
@@ -205,19 +205,19 @@ func VerifyPending(pending *DPCPending,
 	// Check if all the ports in the config are out of pciBack.
 	// If yes, apply config.
 	// If not, wait for all the ports to come out of PCIBack.
-	portInPciBack, portName, usedByUUID := pending.PendDPC.IsAnyPortInPciBack(aa)
+	portInPciBack, portIfName, usedByUUID := pending.PendDPC.IsAnyPortInPciBack(aa)
```

Comment:
if you are changing portName, why not change it to ifname?
Otherwise you have to explain what is the conceptual difference between a portIfName and an ifname,